backend/src/test/java/io/debezium/configserver/util/AbstractInfrastructureTestResourceLifecycleManager.java

```diff
@@ -20,6 +21,10 @@ public Map<String, String> start() {
                 KafkaConnectClientFactory.PROPERTY_KAFKA_CONNECT_URIS,
                 "http://" + Infrastructure.getDebeziumContainer().getHost() + ":" + Infrastructure.getDebeziumContainer().getMappedPort(8083)
         );
+        config.put(
+                JolokiaClient.PROPERTY_JOLOKIA_CLIENT_URI,
```

Comment:
@rk3rn3r If we remove the config property how can we pass the URL into our connector config?

Reply:
I think you switched the main code to use the URI of the selected Kafka Connect cluster (based on the `{{cluster}}` URL parameter? This shouldn't be necessary now, is it? I feel confused now, am I wrong?

Reply:
Since we are using the testcontainers, we need to use the DebeziumContainer's host and mapped port like we do in line 20 here for `kafka.connect.uri`. Otherwise it doesn't work and throws this error :
```
HTTP Request to /api/connectors/1/my-postgres-connector/metrics failed, error id: 5ab3b6c8-5940-4aa2-b9e2-53630fb30504-1: org.jboss.resteasy.spi.UnhandledException: java.lang.RuntimeException: org.jolokia.client.exception.J4pConnectException: Cannot connect to http://localhost:8778/jolokia: Connect to localhost:8778 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused (Connection refused)
	at org.jboss.resteasy.core.ExceptionHandler.handleApplicationException(ExceptionHandler.java:110)
	at org.jboss.resteasy.core.ExceptionHandler.handleException(ExceptionHandler.java:370)
	at org.jboss.resteasy.core.SynchronousDispatcher.writeException(SynchronousDispatcher.java:218)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:519)
	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:261)
	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:161)
	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
	at org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:164)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:247)
	at io.quarkus.resteasy.runtime.standalone.RequestDispatcher.service(RequestDispatcher.java:82)
	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.dispatch(VertxRequestHandler.java:147)
	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler$1.run(VertxRequestHandler.java:93)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:576)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.RuntimeException: org.jolokia.client.exception.J4pConnectException: Cannot connect to http://localhost:8778/jolokia: Connect to localhost:8778 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused (Connection refused)
```

Reply:
@ani-sha I was suggesting on Google Chat that we do a quick call to settle on this.
The only thing that we should be using is https://github.com/debezium/debezium-ui/blob/main/backend/src/test/java/io/debezium/configserver/util/AbstractInfrastructureTestResourceLifecycleManager.java#L19-L22 and we are now using the rewritten URI from the related cluster from the `{{cluster}}` parameter, as we do now here:
https://github.com/debezium/debezium-ui/pull/702/files#diff-b0c80e985e9cdbb002cbbb501120be53968a781acacce6e9bbe55476a333838cR548-R549